### PR TITLE
fix: `npm run build` scripts windows path error

### PR DESCRIPTION
**Pull request by liulinboyi**
Before submitting a new pull request, please make sure:

- [ ] Test cases have been added/updated for the code you will submit.
- [ ] Documentation has added or updated.
- [ ] Commit message is following the [Convention Commit](https://conventionalcommits.org/) guideline.
- [ ] Squash the repeat code commits, short patches are welcome.

fix the issue https://github.com/Tencent/Hippy/issues/39#issuecomment-569850059

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/Hippy?pullRequest=103) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/Hippy?pullRequest=103) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/Hippy?pullRequest=103) it.</sub>

**Comment by liulinboyi** [comment]
> Following codes are updated to `path.resolve()` and `path.basename()`.
> 
> Never process the path with string.

now  i have change the way of process the path.

File: scripts/js2cpp/build.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ const fs = require('fs');
 const path = require('path');
 const readline = require('readline');
 const babel = require("@babel/core");
+const os = require('os');
```
[the remark this answers is not in the document]
Comment (liulinboyi) [reply scripts/js2cpp/build.js]: ok i will remove this line.

File: scripts/js2cpp/build.js [hunk 2]
```diff
@@ -123,8 +123,7 @@ function GenerateCPP(platform, buildDirPath) {
         code += CodePieces.piece_1;
 
         for(let i = 2; i < files_arr.length; i++) {
-            let file_split_arr = files_arr[i].split('/');
-            let file_name = file_split_arr[file_split_arr.length - 1].split('.')[0];
+			  let file_name = path.basename(files_arr[i], ".js")
```
Comment (xuqingkuang) [inline scripts/js2cpp/build.js]: It seems indent have mistaken.

Comment (liulinboyi) [reply scripts/js2cpp/build.js]: ok i have change the indent to the origin codes indent .

**Pull request merged**